pkg/network/ebpf/c/runtime/tracer.c

```diff
@@ -426,7 +446,7 @@ int kprobe__tcp_set_state(struct pt_regs *ctx) {
     }
 
     tcp_stats_t stats = { .state_transitions = (1 << state) };
-    update_tcp_stats(&t, stats);
+    update_tcp_stats(&t, stats, RETRANSMIT_COUNT_INCREMENT);
```

Comment:
This is a bit confusing, because you aren't changing the retransmit count. Can you add a `RETRANSMIT_COUNT_NONE` like how there is a `PACKET_COUNT_NONE` and use it here?